deploy/docker-migrate.sh

```diff
@@ -4,7 +4,5 @@ set -u   # crash on missing env variables
 set -e   # stop on any error
 set -x   # print what we are doing
 
-cd /app/
-
-yes yes | python manage.py migrate --noinput
+python manage.py migrate --noinput
```

Comment:
Voorheen vroeg Django nog steeds om input wanneer er "stale content types" waren, zelfs met de flag `--noinput`. Om die reden was `yes yes` nodig, anders bleef de migration eindeloos hangen op die input. 

Weet jij of dat nu gefixed is in Django? Zo niet, dan is `yes yes` nodig.